### DISCUSSION TITLE
在线编译Pake Cli成功run workflow后显示No File Found

### DISCUSSION THREAD

**Discussion by RichardDuan1** [tw93/Pake#577] [Q&A]
在这个Online-Compilation-(used-by-ordinary-users)模式下，经过Fork和Build App with Pake Cli后（设定如下）
[https://image-1316038659.cos.ap-shanghai.myqcloud.com/1.png?q-sign-algorithm=sha1&q-ak=AKIDQQVFo-7jA2yvjciMmmpGEOFSJq8PCzMAk9lr44Uq2RGvZYGMtOV2eEfe16XUooQl&q-sign-time=1695226762;1695230362&q-key-time=1695226762;1695230362&q-header-list=host&q-url-param-list=ci-process&q-signature=2e67887709cd2237561e6fe4f8e9246bac0235ca&x-cos-security-token=nFmInHZCp41Hu9c5J29gKTbCXnvA2oUaeda213f994618edaf7ca64212b818492pWv2z2ccelGac_yuupz01BRCcq1K2jTTk_4BbU1aH7h_UsiHTVJddqsKoBVW4n4Mkxve0q3fTGgXMNuvqE3bgJ5OVEP46uAPGZ0aV9aRBjmfRNLYo8LqDw1NeLtuWZmRkGJLAjS7ftGpKsyiOwEtK8917yukExgWZ3-53lKpnIEt4Boa-yyuiD9ynSRSOcL0&ci-process=originImage](url)
运行成功，但是在Annotation那里显示如下
[https://image-1316038659.cos.ap-shanghai.myqcloud.com/2.png?q-sign-algorithm=sha1&q-ak=AKID31hnRrotpRoV3LL8-cc58SsKM_8hlYLVx5jKCx111YCaOKkhKBrSuA_0Ol65UD-P&q-sign-time=1695226804;1695230404&q-key-time=1695226804;1695230404&q-header-list=host&q-url-param-list=ci-process&q-signature=272a135b05699ad0af0ed8f79f3e59559e476a55&x-cos-security-token=nFmInHZCp41Hu9c5J29gKTbCXnvA2oUa648cc2817d9b2d53494008df5b95dde1pWv2z2ccelGac_yuupz01IHYwIPAZnglCdogc8A03OTpBNaQYp5_3OKJrMZdlZFo3_dvvw7b8O7bKCfTpNbkEr2jRBeowzpZYDpdeRbGkZqX_3qNuMlnAnErWJttiBb62VNqQrTAopq_eV7nGPZtwBJM_f1ayw6TvVASFEZWSftpMy8jsoX0sFR88UG6diud&ci-process=originImage](url)，no file found，不知道是什么原因，请问有大佬能帮忙看一下吗

**Comment by Tlntin** [comment]
命名问题，不能加空格，包已经生成了，但是他的名字和你想要的名字有差异，导致找不到包。
![image](https://github.com/tw93/Pake/assets/28218658/524cbe3e-731a-4151-bf1f-63c3dcbae0ba)
